packages/pretty-format/src/index.js

```diff
@@ -57,15 +59,15 @@ function printFunction(val, printFunctionName) {
   }
 }
 
-function printSymbol(val) {
+function printSymbol(val: Symbol): string {
   return symbolToString.call(val).replace(SYMBOL_REGEXP, 'Symbol($1)');
 }
 
-function printError(val) {
+function printError(val: Error): string {
   return '[' + errorToString.call(val) + ']';
 }
 
-function printBasicValue(val, printFunctionName, escapeRegex) {
+function printBasicValue(val: any, printFunctionName: boolean, escapeRegex: boolean): StringOrNull {
```

Comment:
> Yes, thank you, in all 8 occurrences?

Yeah, you should try as much as possible to never use `any`. It's as an opt-out from Flow's type checking. You should only use it in scenarios like above where Flow can't do what you need it to.